### DISCUSSION TITLE
Feedback on data sources and APIs

### DISCUSSION THREAD

**Discussion by parha** [LearningArena/yrkesvagledning#31] [General]
Part of project goal - give feedback on quality, missing information, data problems, connectibility, etc

**Comment by parha** [comment]
The response for enrichtextdocuments endpoint for https://jobad-enrichments-api.jobtechdev.se/ does not contain taxonomy ids.

**Reply by parha** [reply]
Actually, what are the "competencies"? They can't be found as skills or keywords in taxonomy ... Why doesn't the response contain concept data?

**Reply by parha** [reply]
... in fact, seems hard to even search for skills from competence words - seems odd that there are competence terms which can't be found in any skill labels?

**Reply by parha** [reply]
In general, using freetext/cv/cover letter to search taxonomy for related concepts seems like it would be a really useful function, especially for skills.

**Comment by KenthJohan** [comment]
**ChatGPT**
Hämta egenskaper från ett jobb

**Promt:**
Vilka egenskaper har man fått från att jobba som flygvärd?

**Svar:**
Att arbeta som flygvärd är en krävande och ansvarsfull roll som kräver en mängd olika färdigheter och egenskaper. Några av de vanligaste egenskaperna som man kan utveckla genom att jobba som flygvärd inkluderar:

1. Kundservice: Flygvärdar är ofta de första personerna som passagerare interagerar med på flyget, och en viktig del av deras roll är att se till att passagerarna känner sig välkomna och bekväma. Att ha starka kundservicefärdigheter, som att vara vänlig, tillmötesgående och lyhörd, är därför avgörande.

2. Kommunikation: Flygvärdar kommunicerar regelbundet med passagerare, kollegor och piloter. De måste kunna uttrycka sig tydligt och korrekt för att förmedla viktig information och säkerhetsinstruktioner på ett effektivt sätt.

3. Stresshantering: Att arbeta som flygvärd kan vara stressigt, särskilt under situationer som förseningar eller kriser. Flygvärdar måste kunna hantera stress och press på ett effektivt sätt för att kunna fortsätta arbeta effektivt och upprätthålla en lugn och professionell framtoning.

4. Teamarbete: Flygvärdar arbetar ofta i team och samarbetar med kollegor för att se till att flyget går smidigt och säkert. Att ha starka teamarbetsfärdigheter, som att vara flexibel, samarbetsvillig och hjälpsam, är därför viktigt.

5. Säkerhetsmedvetenhet: En av de viktigaste rollerna för flygvärdar är att säkerställa att passagerarna följer säkerhetsinstruktionerna och att allt går smidigt och säkert ombord på flygplanet. Att ha en hög medvetenhet om säkerhetsföreskrifter och riskhantering är därför avgörande för att lyckas som flygvärd.

**Comment by parha** [comment]
AF taxomony seems to contain ~400 ssyk-level-4. How well do they cover AF data such as ads?

**Reply by parha** [reply]
SCB said there are ~8500 occupations?

**Reply by parha** [reply]
Having a quality/strength value for relations in the graph would probably be useful. For the PoC in particular, it would be useful when looking at skills related to ssyk-level-4.

**Reply by parha** [reply]
For the PoC, occupation field would be a very useful concept, but only has 12 instances, which then become too general, at least for our use.

**Reply by parha** [reply]
How often is it updated? Would be a more useful resource to follow the quickly changing times, instead of standardized terms which change very slowly.

**Comment by parha** [comment]
AF Yrkesprognoser API? - Coming soon (SCB data access is complex)
Also, more long term and bigger picture, maybe move towards _**skills**_ prognosis?

**Reply by parha** [reply]
Using forecast json file, not all ssyk-level-4 are included.

**Reply by parha** [reply]
Not all ssyk-level-4 included in forecast

**Comment by parha** [comment]
AF JobSearch search results - relevance factor sounds interesting, but always seems to be 1.

**Reply by parha** [reply]
ad data uses key "concept_id" instead of "concept_taxonomy_id" as in several other places

**Reply by parha** [reply]
require more structured data input for af ads? Would be useful to have e.g. skills requirements filled out in jobsearch ad data ... In all career change situations it would be more interesting to have more focus/quality on skills.

**Reply by parha** [reply]
Include forecast (yrkesprognos) data in response?

**Reply by parha** [reply]
In line with PoC, in order to find a new career path, it would have been interesting to be able to use jobsearch search api endpoint's different parameters in combination with excluding occupation names/groups/fields.

**Reply by parha** [reply]
For the different search parameters, could be a bit clearer on when it applies AND or OR operations.

**Reply by parha** [reply]
organisationsnummer?

**Reply by parha** [reply]
For AF to do - If missing parts, e.g. skills, use LLM to fill in data.

**Reply by parha** [reply]
very few skills fields filled in.

**Comment by parha** [comment]
[MYH Kavalifikationsdatabas](https://www.myh.se/kvalifikationer)
API?
Koppling mellan kvalifikation och ssyk?

**Reply by parha** [reply]
Tried searching for ssyk-level-4 labels, seemed to have a very low hit rate ...

**Comment by parha** [comment]
AF to gather data on personality tests / CVs / etc connected to job outcome?

**Comment by parha** [comment]
Is there data looking at job position trajectories, instead of personality evaluations etc.

**Comment by parha** [comment]
For AF's data, we could have used an initial overview connecting taxonomy, data and APIs on a very high "purpose" level.

**Reply by parha** [reply]
Something better than our first attempt: https://gitlab.com/arbetsformedlingen/dy/digital-yrkesvaegledning/-/wikis/PoC-Data-Map
